src/_data/side-nav.yml

```diff
@@ -96,6 +96,8 @@
           permalink: /web
         - title: Get started
           permalink: /web/get-started
+        - title: JS/TS interop
```

Comment:
I'm a little leery of using an abbreviation like "interop" in the sidenav. (Web results mostly link to a particular conference.) Let's do "interoperation" unless you have a better word.

Reply:
Isn't that the term most use? First many hits here are exactly about calling JS from language-x: https://www.google.com/search?q=js+interop

Reply:
Interestingly, there's https://developer.android.com/kotlin/interop, which has **interop** in the title but **interoperability** in the page. Let's follow their lead.